pygenesys/version.py

```diff
@@ -64,5 +64,5 @@
 MICRO = _version_micro
 VERSION = __version__
 PACKAGE_DATA = {'pygenesys': [pjoin('data', '*')]}
-REQUIRES = ["numpy"]
+REQUIRES = ['numpy', 'pandas', 'matplotlib', 'pytest']
 PYTHON_REQUIRES = ">= 3.5"
```

Comment:
To make your life easier and so you don't need to support all of the 3x series you could do >=3.6.